components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -284,4 +296,12 @@ export default {
   opacity: 0;
   position: absolute;
 }
+
+.red-text {
+  color: red;
+}
+
+.red-text a {
+  color: red;
```

Comment:
Double nit: you might want to extract this to a var so you can reuse it later. And maybe even add more text colors like Success or Warn